src/main/resources/application.properties

```diff
@@ -1,9 +1,8 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/kodilla_project?serverTimezone=Europe/Warsaw&useSSL=False
+spring.datasource.url=jdbc:mysql://localhost:3306/kodilla_project?serverTimezone=Europe/Warsaw&useSSL=False&DB_CLOSE_ON_EXIT=FALSE
```

Comment:
czy `DB_CLOSE_ON_EXIT=FALSE` jest konieczne? Prawdopodobnie nie powinno byc tej flagi
https://stackoverflow.com/questions/52480973/h2-in-memory-test-db-closed-despite-db-close-on-exit-false